### ISSUE TITLE
Once expired, let the user know that they are not able to access it due to expiration of the token

### ISSUE THREAD

**Issue by hwangs12** [hwangs12/marketplace_authentication_starter#6]

**Comment by sunyongpark2021** [comment]
Once the token is expired, reset everything and prompt the user to the sign in screen??

**Comment by hwangs12** [comment]
yeah but this again will be implemention in our api. Say a user sign in and we generate them a token which is encrypted. when we decrypt the original message will say like {start: starttime, end: endtime} we can check the user's time is past the _endtime_ and then we can tell the user 'your token has expired' while changing the user attribute logged-in to 'false' at the same time
